lib/std/c/openbsd.zig

```diff
@@ -401,6 +401,8 @@ pub const sockaddr = extern struct {
     };
 };
 
+pub const IFNAMESIZE = 16;
+
```

Comment:
if the c name is IF_NAMESIZE these should be in a struct like the others

Reply:
can u fix freebsd along with this change?

Reply:
- `IF_NAMESIZE` is a standalone'ish identifier. ie: there are no `IF_*` things.
- freebsd and other OS's already have it